src/state/queries/preferences/const.ts

```diff
@@ -20,11 +20,9 @@ export const DEFAULT_THREAD_VIEW_PREFS: ThreadViewPreferences = {
   lab_treeViewEnabled: false,
 }
 
-const DEFAULT_PROD_FEED_PREFIX = (rkey: string) =>
-  `at://did:plc:z72i7hdynmk6r22z27h6tvur/app.bsky.feed.generator/${rkey}`
 export const DEFAULT_PROD_FEEDS = {
-  pinned: [DEFAULT_PROD_FEED_PREFIX('whats-hot')],
-  saved: [DEFAULT_PROD_FEED_PREFIX('whats-hot')],
```

Comment:
I'm not quite ready to pull the trigger on this. We want to be sure the user has a feed; one of my current tasks is to ensure if a new user sees content, and the way we'll do that is by sampling from their saved feeds -- so we want to guarantee that one is set.

Reply:
@pfrazee just want to make sure you see this